core/drivers/scmi-msg/smt.c

```diff
@@ -211,3 +211,17 @@ void scmi_smt_init_agent_channel(struct scmi_msg_channel *chan)
 
 	panic();
 }
+
+void scmi_smt_set_shared_buffer(struct scmi_msg_channel *channel, void *base)
+{
+	assert(!channel->shm_addr.va && !channel->shm_addr.pa);
+
+	channel->shm_addr.va = (vaddr_t)base;
+	channel->shm_addr.pa = virt_to_phys(base);
+}
+
+void scmi_smt_unset_shared_buffer(struct scmi_msg_channel *channel)
```

Comment:
Why not use `scmi_smt_set_shared_buffer(channel, NULL);` instead? That's what this function does -- set addresses to NULL/zero.

Reply:
you're right. thanks :)

Reply:
OK, with this function removed the commit is:
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`